bench/dashboard/competitive.html

```
@@ -329,5 +338,7 @@ <h2>Full Results</h2>
 
 main();
 </script>
+</main>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </main> ]

(tag-pair)